apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/converter/db/TransactionEntityRowMapper.java

```diff
@@ -0,0 +1,68 @@
+/*
+ *  Copyright © 2018-2019 Apollo Foundation
+ */
+
+package com.apollocurrency.aplwallet.apl.core.converter.db;
+
+import com.apollocurrency.aplwallet.apl.core.app.AplException;
+import com.apollocurrency.aplwallet.apl.core.entity.blockchain.TransactionEntity;
+import org.jdbi.v3.core.mapper.RowMapper;
+import org.jdbi.v3.core.statement.StatementContext;
+
+import javax.inject.Singleton;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+
+@Singleton
+public class TransactionEntityRowMapper implements RowMapper<TransactionEntity> {
+
+    @Override
+    public TransactionEntity map(ResultSet rs, StatementContext ctx) throws SQLException {
```

Comment:
SQLException is not impossible here.  We catch SQLException in the method mapWithException and throw RuntimeException. I think it's better to think it over on this case.

Reply:
SQLException is declared in the parent interface RowMapper;